armory/utils/metrics.py

```diff
@@ -68,6 +71,129 @@ def l0(x, x_adv):
     return norm(x, x_adv, 0)
 
 
+class MetricCounter:
```

Comment:
MetricLog?
MetricHistory?

Reply:
Maybe this could be MetricList (since it effectively stores a list of metric values), and I could update the function names to be more similar to list syntax, with append instead of update?